lib/api/src/grpc/proto/points.proto

```diff
@@ -267,6 +268,7 @@ message SearchPoints {
   optional uint64 timeout = 13; // If set, overrides global timeout setting for this request. Unit is seconds.
   optional ShardKeySelector shard_key_selector = 14; // Specify in which shards to look for the points, if not specified - look in all shards
   optional SparseIndices sparse_indices = 15;
+  optional uint32 tokens_count = 16;
```

Comment:
Colbert design document covered this message and `SearchPointGroups`. The user has to send a flattened vector and `SearchPoint`/`SearchPointGroups` have to be unchanged. It's possible to convert from flattened `vector` into colbert vectors using collection DIM